.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: android
 jdk: oraclejdk8
-sudo: false
```

Comment:
Put this back. It causes travis builds to run on their newer container-based infrastructure

http://docs.travis-ci.com/user/workers/container-based-infrastructure/